RedHat-UX/red-hat-design-system
# docs(health-index): swapping PNGs with SVGs

## What I did

1. Swapped out PNGs with SVGs
2. A few of the previous PNGs reference a "Mini" size, which have been corrected with "Small" in the new SVGs


## Testing Instructions

1. Check accuracy of images on [Health Index's docs](https://deploy-preview-1659--red-hat-design-system.netlify.app/elements/health-index/)
